### ISSUE TITLE
[Paper Note] Understanding Data Temporality Impact on Large Language Models Pre-training, Hippolyte Pilchen+, arXiv'26, 2026.05

### ISSUE THREAD

**Issue by AkihikoWatanabe** [AkihikoWatanabe/paper_notes#5608]
# URL
- https://arxiv.org/abs/2605.22769
# Authors
  - Hippolyte Pilchen
  - Romain Fabre
  - Franck Signe Talla
  - Patrick Perez
  - Edouard Grave
# Abstract
  - Large language models (LLMs) are typically trained on shuffled corpora, yielding models whose knowledge is frozen at train time and whose temporal grounding remains poorly understood. In this work, we study the impact of pre-training dynamics on the acquisition of time-sensitive factual knowledge, focusing specifically on data ordering. Our main contributions are twofold. First, we introduce a comprehensive benchmark of over 7,000 temporally grounded questions and an evaluation protocol that enables analysis of whether models correctly associate facts with their corresponding time periods. Second, we pretrain 6B-parameter models on temporally ordered Common Crawl snapshots and compare them against standard shuffled pre-training. Our results show that sequentially trained models match shuffled baselines on general language understanding and common knowledge while consistently exhibiting more up-to-date and temporally precise knowledge. Temporally ordered pre-training yields improved factual freshness, while shuffled pre-training peaks on older data, possibly due to increased factual repetition. These findings, along with the release of our code at https://github.com/kyutai-labs/kairos , checkpoints, and datasets at https://huggingface.co/collections/kyutai/kairos provide a foundation for future research on continual learning for LLMs.
# Translation (by gpt-5-nano)
- 大規模言語モデル（LLMs）は通常、シャッフルされたコーパスで訓練されるため、学習時点で知識が凍結され、時間的根拠づけが十分に理解されていない。本研究では、前訓練のダイナミクスが時間依存的な事実知識の獲得に与える影響を、特にデータの並び順に焦点を当てて検討する。我々の主な貢献は二つある。第一に、時間的根拠を有する質問を7,000件超収録した包括的なベンチマークと、モデルが事実とそれに対応する時期を正しく結び付けられるかを分析できる評価プロトコルを導入する。第二に、6Bパラメータのモデルを時間的に順序付けられたCommon Crawlのスナップショットで事前訓練し、標準のシャッフル訓練と比較する。我々の結果は、時系列的に訓練されたモデルが一般的な言語理解と一般知識においてシャッフル基準と同等以上でありつつ、より最新で時間的により正確な知識を一貫して示すことを示している。時間的順序付けられた前訓練は事実の新鮮さを向上させ、一方でシャッフル訓練はより古いデータでピークに達することが多く、これは事実の反復が増えるためと考えられる。これらの知見とともに、私たちのコードは https://github.com/kyutai-labs/kairos で公開され、チェックポイントおよびデータセットは https://huggingface.co/collections/kyutai/kairos で提供している。これらは、LLMsの継続的学習に関する将来の研究の基盤となる。
# Summary (by gpt-4o-mini)
- 時間的根拠を学ぶためのLLMの訓練におけるデータの並び順の重要性を探求。7,000件を超える時間的質問のベンチマークを作成し、事実と時期の結び付けを評価。6Bパラメータモデルを時系列で訓練した結果、シャッフル訓練と同等以上の性能を示しつつ、最新の知識を一貫して保持。これにより、時間的順序付けが知識の新鮮さを向上させることを明らかにした。関連コードやデータセットも公開し、今後のLLMの継続学習研究に寄与。

**Comment by AkihikoWatanabe** [comment]
元ポスト:https://x.com/rosinality/status/2057730607321432487?s=46&t=Y6UuIHB0Lv0IpmFAjlc2-Q

**Comment by AkihikoWatanabe** [comment]
事前学習時に時系列に応じて並び替えをしたコーパスと、シャッフルしたコーパスの場合、前者の場合freshな知識が必要な質問に対する応答性能が改善する。実験では、Common Crawlのsnapshotの時刻のタイムスタンプに基づいてorderを決定しているようである（2.3説冒頭）。
<img width="628" height="721" alt="Image" src="https://github.com/user-attachments/assets/3fc2ad69-a4c4-44bc-a01f-d3a87ffa2b6f" />

評価のために作成されたQA例が下記で、NBAのバスケチームのコーチのような時間とともに正解が変化するような事実に関する質問によって構成されているようである。これらはwikipediaから特定の年と紐づいた (subject, relation, object) のタプルを抽出することによって生成される。
<img width="1064" height="596" alt="Image" src="https://github.com/user-attachments/assets/b42ff1d2-1ebe-40d5-9d75-2a0acd2ff118" />
